include/seahorn/Expr/Smt/ExprToZ.def

```diff
@@ -0,0 +1,512 @@
+#pragma once
```

Comment:
Why not to move the whole file ExprToZ.def to  lib/seahorn/Smt/ExprToZ.cc?

Reply:
flexibility. You can include `.def` and pay price for templates, or link `ExprToZ` but then you are stuck with particular type for the cache